pkgs/development/python-modules/pytest-watch/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, colorama
+, docopt
+, pytest
+, watchdog
+}:
+
+buildPythonPackage rec {
+
+  pname = "pytest-watch";
+  version = "4.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1fflnd3varpqy8yzcs451n8h7wmjyx1408qdin5p2qdksl1ny4q6";
+  };
+
+  propagatedBuildInputs = [ colorama docopt watchdog pytest ];
```

Comment:
`pytest` should be in `checkInputs` https://nixos.org/nixpkgs/manual/#build-phase.
Otherwise `pytest` is a runtime dep.

Reply:
I put it in `propagatedBuildInputs` because `pytest-watch` is importing it.

https://github.com/joeyespo/pytest-watch/blob/2ec7d59e0cc62722fe343fdccc1e6c8f24710844/pytest_watch/config.py#L5

Reply:
For `pytest`, we usually make an exception and put it in `buildInputs` so that packages using `pytest-watch` can decide whether to use `pytest` or `pytest_3`.

Reply:
@dotlambda made that change in 632271c1b87abab0c0bb87c0c9d2ed78b83fea30 .